_Group Projects/Group_5/program.do

```diff
@@ -0,0 +1,26 @@
+// Set global dir
+
+global wd "/Users/miglepetrauskaite/Desktop/GITHUB/Repo/econ490-fall22/_Group Projects/Group_5"
```

Comment:
I'm not sure you need to set a working directory here! In fact, removing it shouldn't change anything, and it means that I have one less thing to change in order to run your program on my end.

Reply:
Nice catch! Edited this.